scripts/build_docs_config.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
```

Comment:
Did anything change here or did it just move?

Reply:
This is basically the old `scripts/build_docs.py` simplified. Now it's only responsible for creating the `mkdocs.yml` file instead of actually calling the "docstring to markdown" converter (`mathy_pydoc`).